github/repos_releases_test.go

```diff
@@ -44,15 +44,15 @@ func TestRepositoriesService_GetRelease(t *testing.T) {
 
 	mux.HandleFunc("/repos/o/r/releases/1", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		fmt.Fprint(w, `{"id":1}`)
+		fmt.Fprint(w, `{"id":1, "author": { "login": "authorname" }}`)
```

Comment:
Minor style suggestion. The whitespace in the JSON is not consistent with most other places in this repo. When written as a single line, they omit all optional whitespace. See examples:

- https://github.com/google/go-github/blob/3e246d29992beb5a5d2fc253adebea86997c45b0/github/repos_releases_test.go#L264
- https://github.com/google/go-github/blob/3e246d29992beb5a5d2fc253adebea86997c45b0/github/pulls_test.go#L76